tests/behat/features/api/projects/logged_in_user_projects.feature

```diff
@@ -0,0 +1,174 @@
+            "views": 10,
+            "download": 0,
+            "private":false,
+            "flavor": "pocketcode",
+            "uploaded": "REGEX_INT_WILDCARD",
+            "uploaded_string":"more than one year ago",
+            "screenshot_large": "images/default/screenshot.png",
+            "screenshot_small": "images/default/thumbnail.png",
+            "projectUrl": "app/project/1",
+            "downloadUrl": "app/download/1.catrobat",
+            "filesize": 0
+        }
+      ]
+      """
+
+  Scenario: Get logged in user projects with limit = 1
+    Given I use a valid JWT Bearer token for "User1"
+    And I have a request header "HTTP_ACCEPT" with value "application/json"
+    And I request "GET" "/api/projects/user/?limit=1"
+    Then the response status code should be "200"
+    Then I should get the json object:
+      """
+      [
+        {
+            "id": "2",
+            "name":"project 2",
+            "author":"User1",
+            "description":"project 2 description",
+            "version":"0.8.5",
+            "views": 50,
+            "download": 0,
+            "private":false,
+            "flavor": "pocketcode",
+            "uploaded": "REGEX_INT_WILDCARD",
+            "uploaded_string":"more than one year ago",
+            "screenshot_large": "images/default/screenshot.png",
+            "screenshot_small": "images/default/thumbnail.png",
+            "projectUrl": "app/project/2",
+            "downloadUrl": "app/download/2.catrobat",
+            "filesize": 0
+        }
+      ]
+      """
+
+  Scenario: Get logged in user projects with offset = 1
+    Given I use a valid JWT Bearer token for "User2"
+    And I have a request header "HTTP_ACCEPT" with value "application/json"
+    And I request "GET" "/api/projects/user/?offset=1"
+    Then the response status code should be "200"
+    Then I should get the json object:
+      """
+      []
+      """
+
+  Scenario: Get logged in user projects with maxVersion= 0.984
+    Given I use a valid JWT Bearer token for "User1"
+    And I have a request header "HTTP_ACCEPT" with value "application/json"
+    And I request "GET" "/api/projects/user/?maxVersion=0.984"
+    Then the response status code should be "200"
+    Then I should get the json object:
+      """
+      [
+        {
+            "id": "2",
+            "name":"project 2",
+            "author":"User1",
+            "description":"project 2 description",
+            "version":"0.8.5",
+            "views": 50,
+            "download": 0,
+            "private":false,
+            "flavor": "pocketcode",
+            "uploaded": "REGEX_INT_WILDCARD",
+            "uploaded_string":"more than one year ago",
+            "screenshot_large": "images/default/screenshot.png",
+            "screenshot_small": "images/default/thumbnail.png",
+            "projectUrl": "app/project/2",
+            "downloadUrl": "app/download/2.catrobat",
+            "filesize": 0
+        }
+      ]
+      """
```

Comment:
Would be nice to a have a test for the default limit value. Check that only 20 projects will be returned when a user has more projects and no limit value is specified in the request.